packages/zk-transfer-web/src/components/Token.tsx

```diff
@@ -64,6 +88,13 @@ const TokenInput = (props: TokenPropType) => {
   );
 };
 
+export const Skeleton = () => {
```

Comment:
What is the need for a Skeleton react component?

Reply:
That's the animating placeholder that shows up when the Tokens are being fetched.